### ISSUE TITLE
Level 10 is unbeatable

### ISSUE THREAD

**Issue by Rbagman** [reloZid/android-anuto#22]
I've consistently annihilated everything, but the stars, on level 10 with an effective three zone strategy. The stars roll through like an armored convoy.

**Comment by HassanHeydariNasab** [comment]
I'm on Level 13 :D
The game is going to hard untill you resign.
make some variety. for example another set of levels or mods.

**Comment by Almtesh** [comment]
If you can't beat the stars, it's because your turrets don't inflict enough damage. Consider upgrading them.

**Comment by reloZid** [comment]
There are two tips I have concerning the stars:
* Try to place and configure (strategy/locking) the towers in a way so that one star gets focused down after another, so that they have no time to heal themselves.
* Be careful when using slow-down turrets: If the stars come too close to each other they also start healing each other which results in a much bigger healing effect.
I will mark this as help wanted because I think level 10 is definitely beatable.

**Comment by Almtesh** [comment]
You mean that the star heal themselves and the nearby stars at the same time. Well done, that's not very simple to do this.

**Comment by mamins1376** [comment]
I reached 40. here are the keys:
1- find out strategic places.
2- enhance weapens as much as you can, don't fill your field with simple and unenhanced ones.
3- NEVER fight with two enemy groups at the same time. kill one, and then go for the next. i lost level 40 because i didn't know this super simple tip.
4- be creative!

Hope you guys enjoy.

**Comment by mvdan** [comment]
I also get beat by stars, but at level 56. The first turrets at maximum level help as far as focusing fire on one of them goes. The fourth turret at the maximum level helps too if set to "strongest", as they will pull stars with a lot of health back to the beginning of the track and let you deal with weaker ones on their own.

**Comment by mvdan** [comment]
Unless there is something I'm missing, the stars are a little too strong. Or the turrets just aren't balanced against them. I find it hard to reach level 56 without abusing the two turrets mentioned above. I can barely build two rocket launchers or any laser turrets, as they are useless against stars.

**Comment by reloZid** [comment]
Yes I think that's a good idea.

**Comment by mvdan** [comment]
Ah, maybe I'll get past level 56 at last :)

**Comment by mvdan** [comment]
To clarify, it should be that any unit can only be healed by a single star. Otherwise a cluster of stars could heal a non-star unit to full heallth too.

**Comment by reloZid** [comment]
Another option would be that stars can't be healed at all but they heal other units the way as they are doing now. I think this way it would be fairly easy to get rid of them but they remain some sort of "counter" against slow-down turrets. What do you think?

**Comment by mvdan** [comment]
I think that would make star-only waves trivial, like 56. For me at least, it would go from being the hardest level to the easiest one (relative to the ones before it).

If you nerf stars that hard, they should have something else going for them. Otherwise, stars alone are just a much worse type of regular unit.

**Comment by Bahjesaispas** [comment]
Lvl 56 is kinda hard. Sound impossible for me.

**Comment by mxmehl** [comment]
I tried level 56 twice now and it seems to be impossible for me as well. I also disabled the slowing towers since they cluster the stars together and increase their healing power. 

I suggest nerfing the stars to let them only heal themselves, not surrounding units. It seems that at stars-only waves the healing stacks enormeously so even when all turrets focus on one single star, they can barely destroy it because somewhere around there's at least one other star healing it.

**Comment by bart9h** [comment]
I don't think the stars should be made weaker. Until the wave 55 everything goes well, and the game is pretty well balanced. It's just the wave 56 that needs fixing.

To me, it seems like it marks the end of the game, like "Ok, I have beat the game till wave 55, when I reach this ridiculous situation where it's impossible to pass. Game over."

**Comment by Bahjesaispas** [comment]
In fact the lvl 56 is hard but not impossible.

http://hpics.li/3283df5

Game over for me at lvl 71. 

You have to use the black hole and the rocket intelligently.

**Comment by gloryroad1** [comment]
Level 10 can be beat but the double line of spinning stars in level 11 is unstoppable for me.  My augmented layout from level 10 that stops those stars hardly puts a dent in any of them in level 11.  Rather than open a ticket, I though I would add a comment here. . .Are the level 11 stars supposed to be that impervious ?

**Comment by jkufner** [comment]
@gloryroad1 They are not that hard ;) Try different strategy.

**Comment by gloryroad1** [comment]
Thx jkufner.  It finally dawned on me to sell all but lasers and rebuild with lasers. . .then return to a mixture for level 12 because lasers don't cut it there.  A huge lesson learned.

**Comment by notpokey** [comment]
I don't think the game has any balancing issues at all. The game is not about holding off every wave until you win the game. It's about getting the highest score you can against against an infinitely large, and ever strengthening army. In such a game, the bad guys _should_ get exponentially harder as the levels progress. Only by being an exponentially better player can you get an incrementally better score.

Even if the game were "fixed" so that level 10, or 56 or 70 was easier to beat, it would only cause complaints that "level 86 is impossible! The game needs better balance." Also once you have a working strategy, you can get to level 40 or 50 pretty quickly, but it takes longer and longer to beat each level after that. I got lucky with a particular strategy in one of my early games, and got to Wave 72 on level 2 (the S shaped one), but it took HOURS. I don't think I want to get to level 90 or 100. That could take DAYS, if my phone didn't melt first.

Try different strategies, and try to learn to play better to get past very difficult levels. Use the "easy" levels (ie. 1-9) to build a defense specifically designed for the harder levels (ie 10). Instead of requesting easier levels, Take a screenshot of your high score, and brag about it to your friends. Or start a forum thread with a leaderboard.

The game is perfectly balanced in my opinion, its just balanced in favor of the enemy. Resistance is futile. How long can you resist?

**Comment by estemon** [comment]
I've arrived to level 82 and I think there's no option to continue, I think the gap on the pentagon enemies strength between this level and previous is so hard. Before start level 82 I've maintained my 20 lives, and this last wave I've lost 27 lives. I've reached twice this level and I've tried different strategies to fight there. My last chance is to sell all missile turrets and buy only machine guns, with locking weakest enemy, but I'm not sure this will be enough. Maybe lasers? Wow, that's a great game!

**Comment by SilverLuke** [comment]
In my opinion level 82 is quite impossible, I have tried some different strategy but the pentagon enemies have too much life.
![photo_2017-12-08_01-41-14](https://user-images.githubusercontent.com/16280387/33745720-e4772fec-dbb8-11e7-9682-d7eb49b06842.jpg)

**Comment by rkoe** [comment]
At least the current version does not have any balancing issues. With the default layout, I was able to win level 81. I also lost at level 82; but I guess level 82 can also be won with a improved strategy.

All you need is a good strategy and good turret-placement:

- If you don't get past level 10, you probably haven't "Enhance"d your weapons.
  Everything until level 12 can be easily won with just 2 fully-enhanced (but not upgraded) bullet towers.
- Fully enhanced towers have a much better damage/money-ratio than non-enhanced towers.
- Although "projectile towers" seem to have the best damage/money-ratio, they only hit a single enemy.
  But for higher levels, that's not enough. For these, it's cruical to hit serveral enemies with each shot
  (with launch towers (mortars/rockets) and/or laser flash towers).
  Using glue towers can enhance this a lot, teleporters can help further.

I don't have any problems with stars/healers; the most challenging enemies are the fliers, since they cannot be slowed down with glue towers. (Especially in level 68/69 and 82.)

**Comment by gloryroad1** [comment]
Nearly 2 years ago I posted here that level 11 was impossible for me.  Though real enough at the time, that lament was way too premature.  I've gotten beyond that...up to the low 70's.  That advance happened primarily because some change or changes a year or more ago rebalanced the game for the defender (by weakening the stars a bit ??) and made it much more certain to get into the 50's and beyond.  But at that same time the loss of the ability to teleport infinitely made it impossible to grind away at the stars until they are gone.  If that were still in-place, I could get out of the 70's.

Anyway, a while back I made a rough spreadsheet analysis of the cost/benefit per enhancement and per upgrade for all the weapons, focusing on incremental hit/credit and incremental range per enhancement/upgrade.  My analysis, which had to depend on the stated hit values, is rough because I don't know the splash effect some weapons have.  And my analysis is not much suited to the sticky towers at all so those weapons are excluded from further mention.

I believe the benefit per cost in the game can be very unbalanced between the different weapons as they upgrade and for a given weapon, between some of the enhancements and upgrades for that weapon.  And some of the upgrades are outrageously bad deals given the cost and the amount of hit it buys.  Part of the imbalance is that one may not be getting equitable incremetal hit for the relatively large cost of upgrading a fully enhanced weapon.

Again, my analysis is rough because I don't know the full effect of various weapons, especially the launcher and laser, but I think it quantifies something I knew from experience and observation...    ...The laser is the least desirable weapon and the last upgrade of the laser is useless, given it's cost (plus I don't think it bounces like the 2nd laser upgrade so one loses that).  ...The rocket is a super weapon, to be bought ASAP, but it is unfairly expensive, even given it's massive hit.   In fact, the gatlings are what blew me past the 40's and into the low 70's.  They are the best deal, but they wont get one beyond that point.

I was wondering if anyone else has analyzed the weapons and their enhancements, upgrades or positioning ??
